### DISCUSSION TITLE
Face Detection Issue - Feature Request

### DISCUSSION THREAD

**Discussion by R-Nabil** [immich-app/immich#12321] [Feature Request]
Hey,

I'm having issues separating the faces of my 2 kids (below 2 both) which I guess can happen. 

I lowered the maximum detection distance to 0.3 (0.5 wasn't separating my two kids and even though 0.4 split the same people into two, it didn't separate my kids). 

However with 0.3 some people are split over 10 times. 

It is what it is, but could a feature be added so that we could deduplicate people ? So first a very aggressive detection is run (0.3) with loads of duplicated people. And then a people de duplication tool could suggest to merge some based on the 0.5 results. 

That would significantly speed up the post processing of too aggressive detection.

**Comment by bo0tzz** [comment]
#4347
